interfaces/cython/cantera/reactor.pyx

```diff
@@ -911,21 +911,57 @@ cdef class WallBase:
         def __set__(self, double value):
             self.wall.setArea(value)
 
+    @property
+    def vdot3(self):
+        """
+        Get the rate of volumetric change [m^3/s] associated with the wall at the
+        current reactor network time. A positive value corresponds to the left-hand
+        reactor volume increasing, and the right-hand reactor volume decreasing.
+
+        .. versionadded:: 3.0
+        """
+        return self.wall.vdot()
+
     def vdot(self, double t):
         """
         The rate of volumetric change [m^3/s] associated with the wall
         at time ``t``. A positive value corresponds to the left-hand reactor
         volume increasing, and the right-hand reactor volume decreasing.
+
+        .. deprecated:: 3.0
+
+            To be changed after Cantera 3.0; for new behavior, see property ``vdot3``.
         """
+        warnings.warn(
+            "To be changed after Cantera 3.0; for new behavior, see property 'vdot3'.",
+            DeprecationWarning)
         return self.wall.vdot(t)
 
+    @property
+    def qdot3(self):
+        """
```

Comment:
Naming the new property `heat_flow` would also avoid needing to introduce this temporary/immediately deprecated property. I'm less sure what to do about `vdot3`; maybe `expansion_rate`?

Reply:
I like the suggestion. For the sake of symmetric naming, I settled on `expansion_rate` and `heat_rate`.